akka-contrib/src/main/scala/akka/contrib/pattern/TransparentExponentialBackoffSupervisor.scala

```diff
@@ -29,9 +29,9 @@ object TransparentExponentialBackoffSupervisor {
    *       maxBackoff,
    *       (random_delay_factor * calculated_backoff) + calculated_backoff)
    *   }}}
-   * @param decider an [[akka.actor.SupervisorStrategy.Decider]] to specify how the supervisor
+   * @param decider an `Decider` to specify how the supervisor
```

Comment:
Also, this way we loose navigation in the docs, I think I have used `[[a.b.Singleton$]]` previosly to make it work with both javadoc and scaladoc.